.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: Build
+
+on:
+  push:
+    branches: [ master, 3x ]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build - JDK ${{ matrix.java }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [ "8", "11", "17" ]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up JDK ${{ matrix.java }}
+      uses: actions/setup-java@v3
+      with:
+        java-version: ${{ matrix.java }}
+        distribution: 'temurin'
+        cache: maven
+    - name: Build with Maven
+      run: mvn -B -e -Prelease -Dgpg.skip=true clean install
+    - name: Run ITs and coverage
+      if: matrix.java == '11'
+      run: mvn -B -e verify -Pcoveralls
```

Comment:
So `coveralls` is only working on Java 11?

Reply:
Sorry, was on other things.

It's running for one of the JDK given we only publish one report in the PR, I don't see the value in running it for all of them. I chose JDK 11 but it could be any of them really.

Given you merged it already, it probably makes sense to remove the Travis build now, I will create a new PR. Also it will allow us to check Coveralls really work as it's hard to know if the report has been published by Travis or GH Actions.